dynamic.js

```diff
@@ -106,22 +106,20 @@ module.exports = function (fastify, opts, next) {
       swaggerObject.externalDocs = externalDocs
     }
 
-    if (!ref) {
```

Comment:
I've played with code a bit more and found out that we don't need this `if` clause. The `swagger` function is called only once at the very beginning, and the second call does not allow us to trigger it once again. Therefore, we're quite safe to remove it, it should not break any logic.